### DISCUSSION TITLE
Enhancement: Auto-Login with OpenID

### DISCUSSION THREAD

**Discussion by webzit** [danny-avila/LibreChat#4464] [Feature Requests & Suggestions]
### What features would you like to see added?

Suggest implementing an environment variable for auto-login when only one OpenID Provider is available, eliminating the need for the current extra click.

### More details

Currently, users are required to click the "OpenID" button to log in using an OpenID Provider, which adds an extra step to the login process. This extra click is unnecessary when there are no alternative login methods available.

To streamline the login experience for users who only have one OpenID Provider, I propose introducing an environment variable to enable auto-login. This would eliminate the need for users to click the OpenID button, thereby speeding up the login process.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bsu3338** [comment]
What about if only one provider is configured then authentication is automatically redirected to it?
